Assets/Scripts/Visuals/UiBuilder/HierarchyItemAtom.cs

```diff
@@ -15,6 +16,10 @@ public new class Data : Atom.Data
             public TextHandler.TextStyle style;
             public HierarchyItemHandler.UiHierarchyItemActions actions;
             public Action<Vector3> rightClickAction;
+            public Action<GameObject> dropActionAbove;
+            public Action<GameObject> dropActionOn;
+            public Action<GameObject> dropActionBelow;
+            public DclEntity draggedEntity;
```

Comment:
Is this needed in here?

Reply:
This is how i relate the gameobject found by unity (when hovering over it) to the dclentity...